jenkins-ci/README.md

```diff
@@ -0,0 +1,75 @@
+# Jenkins CI
+Jenkins CI deploys OpenShift templates and verifies. This CI also runs functional tests from [jaeger-java-test](https://github.com/Hawkular-QE/jaeger-java-test) repository.
+
+## How to setup Jenkins CI
+
+### Prerequirements
+* Setup [openshift](https://github.com/openshift/origin) with the user `jenkinsci`
+
+#### Setup environment for Jenkins
+Jenkins needs set of permissions to deploy/un-deploy Jaeger on Openshift. The following commands satisfy those requirements.
+
+Login as a `root` user on the OpenShift cluster machine and execute the following commands.
+
+```bash
+oc login -u jenkinsci
+oc new-project jaeger-ci
+oc login -u system:admin
+oc create -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/production/daemonset-admin.yml
+oc adm policy add-role-to-user daemonset-admin jenkinsci -n jaeger-ci
+```
+
+#### Setup Jenkins instance on OpenShift
+We have two flavors of Jenkins for Openshift. 1. `jenkins-persistent`, 2. `jenkins-ephemeral`. In this document, We choose the second option. To know more about Jenkins on Openshift read [this page](https://github.com/openshift/origin/tree/master/examples/jenkins)
+
+We tried Jenkins with default `512Mi`. Works too slow. Hence we would recommend `Memory Limit` to `1024Mi` or above.
+
+##### Installation steps
```

Comment:
@pavolloffay This is one time setup. And I do not know how to set `Memory Limit` on CLI.